packages/firestore/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
   values safely without transactions.
 - [changed] Improved performance when querying over documents that contain
   subcollections.
+- [changed] Prepared the persistence layer to support collection group queries.
+  While this feature is not yet available, all schema changes are included
+  in this release. Once you upgrade, you will not be able to use an older version
+  of the Firestore SDK with persistence enabled.
```

Comment:
NOTE: I'll drop this last sentence for iOS and Android since they should support downgrades.